### PR TITLE
[Task] Allow java-master to be specially handled

### PR DESCRIPTION
**Pull request by nonvirtualthunk**
- Adds in special support for java-master, as well as normal master, for situations in which we are replacing an old java service.

### CODE REVIEW

File: circle/do-ecs-deployment.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ while IFS=: read REPO ECS_SERVICE; do
   # retag ${SPECIFIC_BRANCH}, then deploy based on that tag
   docker tag ${REPO_ROOT}/${REPO}:${CIRCLE_BRANCH} ${REPO_ROOT}/${REPO}:${SPECIFIC_BRANCH}
   docker push ${REPO_ROOT}/${REPO}:${SPECIFIC_BRANCH}
-  if [ "${CIRCLE_BRANCH}" = "master" ]; then
+  if [ "${CIRCLE_BRANCH}" = "master" || "${CIRCLE_BRANCH}" = "java-master" ]; then
```
Comment (jbarnettwomply) [inline circle/do-ecs-deployment.sh]: `man test` says you should use a `-o` instead of a `||`

Comment (nonvirtualthunk) [inline circle/do-ecs-deployment.sh]: oh bash, changed

**Comment by jbarnettwomply** [comment]
+1

**Comment by pklock** [comment]
+1 merging

**Pull request merged**